lib/gui/app/components/file-selector/controllers/file-selector.js

```diff
@@ -41,8 +45,27 @@ module.exports = function (
    * @example
    * FileSelectorController.getFolderConstraint()
    */
-  this.getFolderConstraint = () => {
+  this.getFolderConstraints = utils.memoize(() => {
     // TODO(Shou): get this dynamically from the mountpoint of a specific port in Etcher Pro
+    // FIXME(Shou): splitting on ',' is unsafe
```

Comment:
On Unix, the `$PATH` variable is separated by `:` characters, on Windows the `%PATH%` variable is separated by `;` characters. Might make sense to re-use those semantics in Etcher?

Reply:
We're moving to a JSON config file so this is only a temporary 'hack' given that all `,:;` can appear in a filename leading to some jagged character escaping